vendor/llm2vec_monGARS/setup.py

```
@@ -0,0 +1,44 @@
+from pathlib import Path
+from setuptools import setup, find_packages
+
+package_name = "llm2vec"
+version: dict[str, str] = {}
+with open(Path(__file__).parent / package_name / "version.py", encoding="utf-8") as fp:
+    exec(fp.read(), version)
```

Ruff (0.13.3)

Use of `exec` detected

(S102)